.github/CONTRIBUTING.md

```diff
@@ -67,6 +67,12 @@ pytest tests
 make docs
 ```
 
+
+#### Adding New Dependencies
+
+To add new dependencies to the project, make sure altering the *environment.yml* file. It will automatically update the *requirements-dev.txt* .
```

Comment:
```suggestion
To add new dependencies to the project, make sure to alter the *environment.yml* file. It will automatically update the *requirements-dev.txt* .
```

Reply:
Hi @cosmicBboy ! I added the command and also a small description where to add the dependencies on the setup.py file.